wdfn-server/setup.py

```diff
@@ -61,26 +67,33 @@ def identify_data_files(directory_names, exclusions=('.gitignore', '.webassets-c
                            for ex in exclusions)
             ]
             if pathnames:
-                data_file_element = (root, pathnames)
+                data_file_element = (installation_directory, pathnames)
                 directory_data_files.append(data_file_element)
     return directory_data_files
 
 
-setup(name='usgs_waterdata_ui',
-      version='0.8.0dev',
-      description='USGS Water Data',
-      author='Mary Bucknell, Andrew Yan, Dan Naab, Janell Fry',
-      author_email='mbucknell@usgs.gov',
-      packages=find_packages(),
-      include_package_data=True,
-      long_description=read('README.md'),
-      install_requires=read_requirements()['install_requires'],
-      platforms='any',
-      test_suite='unittest:TestLoader',
-      zip_safe=False,
-      # include the tier agnostic configuration file in the distributable
-      # the file gets placed in site-packages upon dist installation
-      py_modules=['config'],
-      # include static files in the distributable
-      # they will appear in the root of the virtualenv upon dist installation
-      data_files=identify_data_files(['assets/dist', 'data']))
+setup(
+    name='usgs_waterdata_ui',
+    version='0.8.0dev',
+    description='USGS Water Data',
+    author='Mary Bucknell, Andrew Yan, Dan Naab, Janell Fry, Aaron Briggs',
+    author_email='mbucknell@usgs.gov',
+    packages=find_packages(),
+    include_package_data=True,
+    long_description=read('README.md'),
+    install_requires=read_requirements()['install_requires'],
+    platforms='any',
+    test_suite='unittest:TestLoader',
+    zip_safe=False,
+    # include the tier agnostic configuration file in the distributable
+    # the file gets placed in site-packages upon dist installation
+    py_modules=['config'],
+    # include static files in the distributable
+    # they will appear in the root of the virtualenv upon dist installation
+    data_files=identify_data_files(
+        [
+            ('assets', '../assets/dist'),
```

Comment:
Would this also require a change in the static symlink from './assets/dist` to `./assets`?

Reply:
I'm not sure I follow. Where is that `./assets/dist` symlink?

Reply:
This one: https://code.usgs.gov/WDFN/waterdataui_config/blob/master/roles/application/tasks/deploy.yml#L50-54